### ISSUE TITLE
Feature Request: "Maybe" option

### ISSUE THREAD

**Issue by bluedawg80** [lukevella/Rallly#47]
It would be nice to have the ability to Mark a date as tentative or maybe. Perhaps it could be a yellow check mark instead of green if you click the box twice. Multiple users for my polls have requested it.

**Comment by erikogan** [comment]
I agree that having a Maybe (better, to me, would be “If Need Be”) option would be a very welcome addition. It’s one of the few reasons I cannot convince our group to switch from Doodle.

@lukevella I’m debating diving in and seeing if I can offer this as a PR. Here’s my current thinking: make it optional (and not enabled by default) on a . If enabled, when selecting dates the first click on a checkbox becomes “Yes” (indicated with a green background?), the second becomes “Maybe / If Need Be” (yellow background?) and third deselects.

Would you object to that? I don’t know how invasive what I’m proposing might be, and I’d hate to do the work if it won’t be accepted.

**Comment by deadlyicon** [comment]
👍 yes please

**Comment by hairykiwi** [comment]
@erikogan I agree with your proposed cyclical-style interface.

Whether titled "Maybe" or "If Need Be" could even be configured by the event organiser - or at least consider adding both options. By comparison, WhenAvailable.com's 3-option interface is a slow UI experience for users marking up more than a few days.
